packages/client/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
```

Comment:
Should we be using the `compilerOptions` from Gutenberg? Or at least aligning where it makes sense?

E.g. there's no reason to have a lower target/module

https://github.com/WordPress/gutenberg/blob/trunk/tsconfig.base.json